internal/trigger/messagebus/messaging_test.go

```diff
@@ -212,15 +213,16 @@ func TestInitializeAndProcessEventWithOutput(t *testing.T) {
 
 	transform1 := func(edgexcontext *appcontext.Context, params ...interface{}) (bool, interface{}) {
```

Comment:
use new AppFunction type?

Reply:
Can't use it here when defining the actual function, only when using as a Type